go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/skupperproject/skupper
 
-go 1.19
+go 1.23
```

Comment:
suggest go, toolchain as used here:

https://github.com/skupperproject/skupper/blob/96648003e55932aff63fec301e8da8d700b15f20/go.mod#L3

Reply:
You are right, thanks!